### PR TITLE
read_carroucell

### PR DESCRIPTION
**Pull request by atravert**
Add data  and test for read_carroucell  (issue #185)

### CODE REVIEW

**Comment by fernandezc** [comment]
HI Arnaud,
I think you should reset your change on make.py to the current master which work well.

**Comment by fernandezc** [comment]
Also I would like you to add lines in changeling.md at the root describing succinctly the changes added.

```
# What\'s new

## Version 0.2.7 ....

* xxxxxxx
```

**Comment by atravert** [comment]
Ok so for instance in the case of read_caroucell, I would add this ?:

## Version 0.2.7 **TASKS** **NEW FEATURES** * Test for read_carroucell()

**BUGS FIXED** * Fix a problem with read_carroucell()

Le 03/02/2021 à 09:36, Christian Fernandez a écrit :

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +21,8 @@
 
 * Compatibility with python 3.6
 
+* Fix a problem with read_carroucell()
```
Comment (fernandezc) [inline CHANGELOG.md:24]: You should put this in a new section (0.2.7) as 0.2.6 is already published.

File: tests/test_readers_writers/test_read_carroucell.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# This is a sample Python script.
+
```
Comment (fernandezc) [inline tests/test_readers_writers/test_read_carroucell.py:2]: Just remove this line above and its perfect!

**Review by fernandezc — CHANGES_REQUESTED** [review]

File: tests/test_readers_writers/test_read_carroucell.py (hunk 2 above)
Comment (atravert) [reply tests/test_readers_writers/test_read_carroucell.py]: done !

File: CHANGELOG.md (hunk 1 above)
Comment (atravert) [reply CHANGELOG.md]: done !

**Review by fernandezc — APPROVED** [review]
Perfect!  I'll proceed with the merge

**Pull request merged**